frontend/test/tickets/tickets.sagas.spec.ts

```diff
@@ -125,6 +219,18 @@ describe('Tickets: sagas', () => {
 					.put.like(alertAction('trying to fetch templates'))
 					.silentRun();
 			})
+
+			it('should call fetchContainerTemplate  detail endpoint', async () => {
+				const template = templateMockFactory();
+				mockServer
+					.get(`/teamspaces/${teamspace}/projects/${projectId}/federations/${modelId}/tickets/templates/${template._id}`)
+					.reply(200, template);
+	
+				await expectSaga(TicketsSaga.default)
+					.dispatch(TicketsActions.fetchTemplate(teamspace, projectId, modelId, template._id, true))
+					.put(TicketsActions.replaceTemplateSuccess(modelId, template))
+					.silentRun();
+			})
 		})
```

Comment:
You didn't add test cases for 404 handling, is that alright?

Reply:
I dont care a lot about that in this case TBH